submissions/rubyAdeniregun/php/listview.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+require_once("db_connection.php");
+
+echo "<h5> List View Page </h5>";
+
+try{
+    $sql= "SELECT * FROM customer_details";
+    $result= $pdo->query($sql);
+    if ($result->rowCount() > 0){
```

Comment:
Now there is progress

But if I reload this `listview.php` this query (`"SELECT * FROM customer_details"`) will always run. But, that is not what we want. We don't want to always run this query all the time, then store it in a cookie. 

Now. when we reload the `listview.php`, we want `$result` to be that coming from that stored in the cookie as long as the cookie is set. If cookie has expired, then `$result` should be that comming from the query

Reply:
The idea is to store the `$result` gotten from the query in a cookie. This action should be within a function.

Now. when we reload the `listview.php`, we want `$result` to be that coming from that stored in the cookie as long as the cookie is set. If the cookie has expired, then `$result` should be that coming from the query